netbox_netprod_importer/__main__.py

```diff
@@ -97,6 +104,23 @@ def parse_args():
             if not isinstance(numeric_level, int):
                 raise ValueError('Invalid log level: %s' % args.verbose)
             logging.getLogger().setLevel(numeric_level)
+
+        args_h = vars(args)
```

Comment:
Prefer `args_h = args.copy()`. Also, maybe rename `args_h` for something more explicit, like `action_params`.

Reply:
Redid, wrong work with Namespace